src/sundials/sundials_mpi_errors.c

```diff
@@ -18,7 +18,7 @@
 #include <sundials/sundials_mpi_types.h>
 #include <unistd.h>
 
-static inline char* combineFileAndLine(int line, const char* file)
+static inline char* sunCombineFileAndLine(int line, const char* file)
```

Comment:
Why not use the version from `sundials_utils.h`?

Reply:
It should use it